benefit/data/sheets/CaseEventToFields/CaseEventToFields-posthearing-nonprod.json

```diff
@@ -169,5 +169,33 @@
     "PageID": "Request_Staging",
     "PageLabel": "Request Document",
     "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "postHearingRequest",
+    "CaseFieldID": "correction",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "COMPLEX",
+    "LiveFrom": "09/03/2023",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "Correction",
+    "PageLabel": "Correction",
+    "PageShowCondition": "postHearingRequestType=\"correction\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CallBackURLMidEvent": "${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEvent?pageId=generateDocument",
+    "CaseEventID": "postHearingRequest",
+    "CaseFieldID": "bodyContentCorrection",
```

Comment:
Same as comment in common changes, is it worth prefixing the post hearing type here and in other places where this is declared (for any other future corrections fields)? Fine if we don't expect any more.

Reply:
Will consider renaming on a future branch if necessary. We have agreed that this syntax is fine here for now.